tool/debian/singa-cuda8.0-cudnn5/DEBIAN/control

```diff
@@ -0,0 +1,14 @@
+Package: singa-cuda8
+Version: 1.0.1
```

Comment:
This number is updated with your build.sh during the building process with the correct version.